packages/services/src/kernel/kernel.ts

```diff
@@ -707,6 +737,21 @@ export interface IManager extends IBaseManager {
    */
   readonly runningCount: number;
 
+  /**
+   * Whether comm messages should be sent to kernel subshells, if the
+   * kernel supports it.
+   *
+   * #### Notes
+   * Sending comm messages over subshells allows processing comms whilst
+   * processing execute-request on the "main shell". This prevents blocking
+   * comm processing.
+   * Options are:
+   * - disabled: not using subshells
+   * - one subshell per comm-target (default)
+   * - one subshell per comm (can lead to issues if creating many comms)
+   */
+  commsOverSubshells: CommsOverSubshells;
```

Comment:
Can this be optional?


```suggestion
  commsOverSubshells?: CommsOverSubshells;
```